### ISSUE TITLE
Change membership in model.biblPart

### ISSUE THREAD

**Issue by pe-ro** [music-encoding/music-encoding#1351]
The current members of model.biblPart are:
- biblScope
- creation
- genre
- imprint
- perfDuration
- physLoc
- recipient
- relatedItem
- series
- textLang
- editionLike
- respLike

To encourage the use of `<biblFull>` (mentioned in #1078), I propose to remove 
- creation
- perfDuration
- physLoc

and add
- repository

`<creation>` and `<perfDuration>` are not part of the usual bibliographic *reference* which `<bibl>` is designed to capture.  If these data points are needed, then the encoder should use `<biblStruct>` or `<biblFull>` which allow a fuller bibliographic *description*. `<physLoc>` will be replaced by `<repository>`.  The `<identifier>` child of `<physLoc>` is available in `<repository>`, but the `<head>` and `<history>` children will be lost.  However, I don't think the impact will be very great.

PR to follow soon.

**Comment by riedde** [comment]
@pe-ro Is this still relevant, after #1367 has been merged?

**Comment by pe-ro** [comment]
@riedde, this is no longer relevant.
